lib/test/cmaf/ham/init.ts

```diff
@@ -0,0 +1,34 @@
+import { setDashParser, setDashSerializer, setHlsParser, type DashManifest, type HlsManifest } from '@svta/common-media-library';
+import { Parser } from 'm3u8-parser';
+import { Builder, parseString } from 'xml2js';
+
+setDashParser((raw: string) => {
+	let parsed: DashManifest;
+	parseString(raw, (err: Error | null, result: DashManifest) => {
+		if (err) {
+			throw new Error(err.message);
+		}
+		parsed = result as DashManifest;
+	});
+	// @ts-ignore
+	return parsed;
+});
+
+setHlsParser((text: string) => {
+	const parser = new Parser();
+
+	parser.push(text);
+	parser.end();
+	const parsedHlsManifest = parser.manifest as any;
```

Comment:
Is it worth moving the type here, so the return statement will be without type assertion?

```suggestion
	const parsedHlsManifest = parser.manifest as HlsManifest | undefined;
```

Reply:
That change won't compile: `Conversion of type Manifest to type HlsManfiest may be a mistake because neither type sufficiently overlaps with the other. If this was intentional, convert the expression to unknown first.`.

Reply:
Added a comment describing the need for the forced casting.